test/integration/bootstrap/bootstrap_test.go

```diff
@@ -15,20 +15,113 @@
 package bootstrap
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
 	"github.com/stretchr/testify/assert"
+	"github.com/terraform-google-modules/terraform-example-foundation/test/integration/testutils"
```

Comment:
IMHO the upstream isn't expecting external use, so I'd suggest we include the required function in this repo to minimize any future compatibility issues.

Reply:
We are planning to move this function (and others we have on foundation) to this repo you mention.
We just didn't do yet to not block the PRs here.
But ASAP we will move all utils to toolkit.